### ISSUE TITLE
roster_tools_test.UT_ReadRosterFromFile.test_05_unknown_teacher FAILS

### ISSUE THREAD

**Issue by djfkahn** [djfkahn/MemberHubDirectoryTools#101]
```
WHERE:
	 test_05_unknown_teacher (roster_tools_test.UT_ReadRosterFromFile)
WHAT:
 Traceback (most recent call last):
  File "/Users/djfkahn/Desktop/DEV/MemberHubDirectoryTools/TEST/roster_tools_test.py", line 45, in test_05_unknown_teacher
    self.assertEqual(2, len(result[0].adults))
AssertionError: 2 != 0

Stdout:
Elementry school student from Roster found with unknown teacher.
('Smith', 'John0', 0, 'Jerry and Jane Smith', 'Unknown Teacher')
1 students processed 1 families.
```

**Comment by djfkahn** [comment]
This was unit test error only.  Because the teacher name is unknown, the entry should be rejected, and the family's adults and children should be empty lists.

Changed the assertions to the following:
```
        self.assertEqual(0, len(result[0].adults))
        self.assertEqual(0, len(result[0].children))
```

**Comment by djfkahn** [comment]
addressed in commit 841d6d9
